amundsen_application/static/css/_layouts.scss

```diff
@@ -74,42 +74,55 @@ $resource-header-height: 84px;
     height: calc(100% - #{$resource-header-height});
 
     > .left-panel {
+      border-right: 4px solid $divider;
+      flex-basis: 600px;
+      flex-shrink: 0;
+      overflow-y: scroll;
+      padding: 0 24px;
+
       > .banner {
         border: 1px solid $stroke;
         height: 40px;
         margin: 24px 24px 0 24px;
         padding: 8px;
       }
-
-      border-right: 4px solid $divider;
-      flex-basis: 634px; // 630 + 4px border
-      flex-shrink: 0;
-      overflow-y: scroll;
-      padding: 0 24px;
     }
 
     > .right-panel {
-      flex-basis: 414px;
+      flex-basis: 500px;
       flex-grow: 1;
       flex-shrink: 0;
       overflow-y: scroll;
       width: 0; // Required for text truncation
     }
+
```

Comment:
do we know what size user's monitors are?

Reply:
<img width="369" alt="Screen Shot 2020-02-24 at 5 26 37 PM" src="https://user-images.githubusercontent.com/4338702/75206266-df034d80-572a-11ea-9f1c-6f02f8ec7a67.png">

About 2% of users are using small or narrow screens. Apart from that some users will put Amundsen on just part of the screen. I've put in a minimum width of 1048 since the layout completely breaks below that.

Reply:
cool! I was just curious how you picked the sizing.

Reply:
Oh, the breakpoints were picked by Wiebke.